content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Configuring Gateway Network Topology (Experimental)
+description: How to configure gateway network topology (experimental).
```

Comment:
instead of experimental, consider using (development) as per feature status pages. Documenting in-development work is fine by me, and nothing in our governance documentation prohibits it. I think documentation before alpha is good practice, although the terminology should match.

Reply:
Done.